docs/guides/chip_tool_guide.md

```diff
@@ -901,8 +938,9 @@ In this command:
 $ ./chip-tool pairing open-commissioning-window 1 1 300 1000 2365
 ```
 
-Note manual pairing code or QR code payload printed in the command output, as it
-will be required by the second Matter admin to join Matter device to its fabric.
+> **Note:** manual pairing code or QR code payload printed in the command
```

Comment:
```suggestion
> Write down the manual pairing code or QR code payload printed in the command
```

As in, this is not a "Note".  This is an instruction for a step you need to take to make this whole thing work.

Reply:
Will rephrase.